modules-js/react-fleet/src/utilities/constants.ts

```diff
@@ -1,17 +1,38 @@
 export const PUBLIC_CSS_URL = 'https://patterns.boston.gov/css/public.css';
 
+export const WHITE = '#fff';
+export const BLACK = '#000';
 export const YELLOW = '#fcb61a';
 export const GREEN = '#62A744';
 export const GRAY_000 = '#f3f3f3';
 export const GRAY_100 = '#e0e0e0';
 export const GRAY_200 = '#C8C8C8';
 export const GRAY_300 = '#828282';
-export const OPTIMISTIC_BLUE = '#288be4';
 export const CHARLES_BLUE = '#091f2f';
-export const FREEDOM_RED = '#fb4d42';
+
+// 2/2019: DARK variations of OPTIMISTIC_BLUE and FREEDOM_RED have been added
+// to provide stronger contrast and readability as text on a light background.
+// The original colors have been renamed as the LIGHT variants.
+
+// LIGHT should always be used for text links on Charles Blue or black. It’s
+// also the preferred variant for some elements’ hover background color, and can
+// also be used in decorative cases.
+//
+// DARK should always be used for text on white or GRAY_000.
+export const OPTIMISTIC_BLUE_LIGHT = '#288be4';
+export const OPTIMISTIC_BLUE_DARK = '#1979cc';
+
+export const FREEDOM_RED_LIGHT = '#fb4d42';
+export const FREEDOM_RED_DARK = '#d22d23';
+
+export const OPTIMISTIC_BLUE = OPTIMISTIC_BLUE_DARK;
+export const FREEDOM_RED = FREEDOM_RED_DARK;
```

Comment:
Are these still needed? Maybe delete them?

Reply:
Good call, thanks for the catch 💯